purview/models/koji.py

```diff
@@ -0,0 +1,55 @@
+from neomodel import (
+    StructuredNode, StringProperty, IntegerProperty, UniqueIdProperty,
+    BooleanProperty, DateTimeProperty, FloatProperty, RelationshipTo)
+
+
+class KojiBuild(StructuredNode):
+    id_ = UniqueIdProperty(db_property='id')
+    tasks = RelationshipTo('KojiTask', 'TRIGGERS')
+    creation_event_id = IntegerProperty(required=True)
+    volume_id = IntegerProperty()
+    owner_id = IntegerProperty()
+    package_id = IntegerProperty(required=True)
+    task_id = IntegerProperty(required=True)
```

Comment:
I don't know if this is necessary since we have the relationship to the tasks through the `tasks` property.